### PR TITLE
Erase if ( i < offset ) { continue;} at start in for loops

### PR DESCRIPTION
**Pull request by theludovyc**
Hello =) !

I work on https://github.com/theludovyc/Godot_STL and I found this ^^ !

### CODE REVIEW

**Comment by akien-mga** [comment]
What happens if `offset` is negative?

File: scene/gui/tabs.cpp [hunk 1]
```diff
@@ -184,10 +184,7 @@ void Tabs::_gui_input(const Ref<InputEvent> &p_event) {
 			}
 
 			int found = -1;
-			for (int i = 0; i < tabs.size(); i++) {
```
Comment (groud) [inline scene/gui/tabs.cpp:187]: Could you check if offset can be negative ? If so, it's possible that this creates a different behavior.
In such case, the condition would be replaced by:
```
for (int i = max(0, offset); i < tabs.size(); i++) {
```
to keep the same behavior.

Comment (theludovyc) [reply scene/gui/tabs.cpp:187]: https://github.com/godotengine/godot/blob/master/scene/gui/tabs.cpp#L122

https://github.com/godotengine/godot/blob/master/scene/gui/tabs.cpp#L179

https://github.com/godotengine/godot/blob/master/scene/gui/tabs.cpp#L862

**Comment by theludovyc** [comment]
https://github.com/godotengine/godot/blob/master/scene/gui/tabs.cpp#L122

https://github.com/godotengine/godot/blob/master/scene/gui/tabs.cpp#L179

https://github.com/godotengine/godot/blob/master/scene/gui/tabs.cpp#L862

Can't be negative

**Comment by groud** [comment]
Ok, then it looks good to me.

**Comment by akien-mga** [comment]
Thanks!

**Comment by theludovyc** [comment]
With pleasure 😊

**Pull request merged**